### DISCUSSION TITLE
How to store very large reconciliation?

### DISCUSSION THREAD

**Discussion by bobobo1618** [firefly-iii/firefly-iii#7967] [Firefly III] [answered]
I've imported my bank's camt.053 statement and would like to store the reconciliation for the history from opening until now.

Opening the reconciliation screen works just fine, marking the transactions works fine (and shows a zero amount for the reconciliation) but clicking the "store reconciliation" button fails. The browser development tools show a 414 response for the GET request to `/accounts/reconcile/71/overview/2019-01-01/2022-12-31...`

I'm using the latest Docker images.

My suggestion is that instead of using GET, the request should use POST?

**Comment by JC5** [comment, marked as answer]
I'm aware of the issue but this is not easily changed right now. You'll have to use shorter time periods.

**Reply by bobobo1618** [reply]
I thought it was impossible because I hadn't kept my statements but turns out my bank lets me generate them as I wish, so I have all the granularity I want and managed to reconcile everything.
